static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -426,3 +426,16 @@ $video-width-desktop: 700px;
     bottom: 2px;
     color: $neutral-3;
 }
+
+.fc-item__video-container {
+    .youtube-media-atom {
+        z-index: 1;
+    }
+
+    .youtube-media-atom__iframe:not(.youtube__video-ready) {
+        ~ .youtube-media-atom__overlay {
+            visibility: hidden;
+            opacity: 0;       
```

Comment:
@duarte no that's fine, I've tested and 1 to 0 works fine and 0 to 1 on video end too, we just don't need that initial 0.